tests/tasks/check_daemon_config_files.yml

```diff
@@ -19,9 +19,8 @@
     that: __rsyslog_conf_stat.stat.size > 1000
   when: __conf_size == "more"
 
-- name: Check file counts in rsyslog.d
-  assert:
-    that: __rsyslog_d_file_count.matched >= __conf_count
+- name: debug
```

Comment:
You've left 2 debugs, is that intentional?

Reply:
yes - this is a WIP - not ready to merge - trying to understand why this is failing on el7